apps/demo-game/src/globals.css

```
@@ -1,17 +1,51 @@
+/*
+ * design-system v4 ships a restrictive package `exports` map that does not
+ * expose the compiled CSS, so the bare specifier
+ * `@uzh-bf/design-system/dist/design-system.css` fails to resolve. Reach the
+ * file directly through node_modules until v5 exports it. Path is relative to
+ * this stylesheet (src/), resolves through the pnpm symlink.
+ */
+@import '../node_modules/@uzh-bf/design-system/dist/design-system.css';
 @import '@gbl-uzh/ui/dist/style.css';
 
-@import 'tailwindcss/components';
+@plugin '@tailwindcss/typography';
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@plugin`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)